packages/gatsby-plugin-treat-example/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "gatsby-plugin-treat-example",
+  "private": true,
+  "version": "0.1.0",
+  "devDependencies": {
+    "babel-preset-gatsby": "^0.2.26",
+    "gatsby": "^2.18.11",
+    "gatsby-plugin-treat": "^0.1.0"
+  },
+  "scripts": {
+    "clean": "gatsby clean && rm -rf node_modules/.cache",
+    "start": "gatsby develop",
+    "build": "gatsby build",
```

Comment:
This is causing a race condition in CI with the other `build` scripts, I'd rename it to `build-site` or similar to avoid it.

Reply:
I've added `treat` as a dependency of `gatsby-plugin-treat-example` (as it should be) which should make `lerna` visit the workspaces in the correct order.

The tests are passing but the `jest` process is hanging waiting for resources to cleanup. Unfortunately I can't reproduce it locally. Will have a bit more of a play.